src/components/welcomeMessage.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <md-card v-if="isFirstView" class="card-content" id="welcomeCard">
+    <md-card-header>
+      <h2 id="welcomeMessage" class="md-title">{{$lang.welcome.message}}</h2>
+    </md-card-header>
+  </md-card>
+</template>
+
+<script>
+  export default {
+    name: 'welcomeMessage',
+    data() {
+      return {
+        isFirstView: true
+      }
+    },
+    created() {
```

Comment:
consider using [beforeCreate](https://vuejs.org/v2/api/#beforeCreate) to improve performance